libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts

```diff
@@ -42,6 +42,29 @@ export const parseJsonOperation = {
   },
 };
 
+export const chunktext = {
+  name: 'chunktext',
+  id: 'chunkText',
+  type: 'Chunktext',
```

Comment:
I would suggest to keep type similar to OperationType and id similar to name or follow what casing is used in other builtins